packages/shared/src/graphql/squads.ts

```diff
@@ -194,6 +203,17 @@ export interface SquadEdgesData {
   sourceMembers: Connection<SourceMember>;
 }
 
+interface UpdateSquadMemberRoleArgs {
```

Comment:
Why do we opt Args here, think most codebase uses Props right?
Just want to align the codebase where possible.

Reply:
Too much BE work 😅 
But since we are in apps repo, I agree, let's keep it to props.